statsmodels/graphics/tests/test_agreement.py

```diff
@@ -7,7 +7,7 @@
 
 try:
     import matplotlib.pyplot as plt
-except ImportError:
```

Comment:
You should add except ImportError to coverage.  This should be skipped everywhere.

Reply:
Agreed.